### DISCUSSION TITLE
Need to write a wiki page that lists all supported boards and  peripherals

### DISCUSSION THREAD

**Discussion by tigert** [MobiFlight/MobiFlight-Connector#1329] [Ideas]
It would be useful to have a wiki page with the officially supported boards listed, with their specs and the relevant information related to Mobiflight: what devices and how many are supported simultaneusly, how many in chained (i2c), and each device type would have their own wiki sub-page that explains wiring and links to relevant additional docs. This kinda overlaps with the website docs, but we could maybe have a separate wiki for official docs in addition to the community wiki, and use the official wiki as main docs, as it is easier to edit.

I could look into this.

**Comment by DocMoebiuz** [comment]
https://github.com/MobiFlight/MobiFlight-Connector/wiki/Input-and-Output-devices

what about this page?
